app/controllers/users_controller.rb

```diff
@@ -22,6 +22,13 @@ def update
     end
   end
 
+  #GET /user/me
+  def auth_user_informations
+    user = User.includes(:course, :categories).find(auth_user.id)
```

Comment:
Chiamando ```auth_user``` abbiamo già l'utente autenticato.
In questo modo stiamo facendo una find su una find della stessa persona.
Basterebbe:
```auth_user.includes(:course, :categories)``` che è da vedere se funziona però.

Reply:
testato e non funge

Reply:
Allora andrei di:
```User.includes(:course, :categories).find_by(id: request.env['jwt.payload']['user_id'])```

Reply:
fato